### PR TITLE
Creation of tests for test driven development

### PR DESCRIPTION
**Pull request by Ragsomar**

### CODE REVIEW

File: test/profiles.http [hunk 1]
```diff
@@ -0,0 +1,25 @@
+GET http://localhost3000/profiles
```
Comment (rocambille) [inline test/profiles.http:2]: this deserves a comment to know what this request does :)

File: test/profiles.http [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+###
```
Comment (rocambille) [inline test/profiles.http]: this deserves a true comment (not just "###") to know what this request does :)

File: test/profiles.http [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+POST http://localhost3000/profiles
+Content-Type: application/json
+{
+  nickname: "XxDarkSasukexX",
+  firstname: "Killian",
+  lastname: "Lelouch"
+}
+
+###
```
Comment (rocambille) [inline test/profiles.http]: this deserves a true comment (not just "###") to know what this request does :)

File: test/profiles.http [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+PUT http://localhost3000/profiles/1
+Content-Type: application/json
+{
+  nickname: "Asomar",
+  firstname: "Pierre-Alban",
+  lastname: "Demars"
+}
+
+###
```
Comment (rocambille) [inline test/profiles.http]: this deserves a true comment (not just "###") to know what this request does :)

File: test/tag.http [hunk 5]
```diff
@@ -0,0 +1,25 @@
+GET http://localhost3000/tags
+
+###
```
Comment (rocambille) [inline test/tag.http]: this deserves a true comment (not just "###") to know what this request does :)

File: test/tag.http [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+POST http://localhost3000/tags
+Content-Type: application/json
+{
+  instrument: "flute de Pan"
+  status: "Amateur",
+  style: "Alternatif"
+}
+
+###
```
Comment (rocambille) [inline test/tag.http]: this deserves a true comment (not just "###") to know what this request does :)

File: test/tag.http [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+PUT http://localhost3000/tags/1
+Content-Type: application/json
+{
+  instrument: "Guitare"
+  status: "Pro",
+  style: "Rock 60's"
+}
+
+###
```
Comment (rocambille) [inline test/tag.http]: this deserves a true comment (not just "###") to know what this request does :)

File: test/home.http [hunk 8]
```diff
@@ -0,0 +1,4 @@
+GET http://localhost:3000/
+
+###
```
Comment (rocambille) [inline test/home.http]: why this line of comment?

File: test/home.http [hunk 9]
```diff
@@ -0,0 +1,4 @@
+GET http://localhost:3000/
```
Comment (rocambille) [inline test/home.http:2]: this deserves a comment to know what this request does :)

File: test/tag.http [hunk 10]
```diff
@@ -0,0 +1,25 @@
+GET http://localhost3000/tags
```
Comment (rocambille) [inline test/tag.http:2]: this deserves a comment to know what this request does :)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: test/tag.http (hunk 10 above)
Comment (ghost) [reply test/tag.http:2]: Fixed, now has a comment.

File: test/home.http (hunk 9 above)
Comment (ghost) [reply test/home.http:2]: Fixed, now has a comment.

File: test/home.http (hunk 8 above)
Comment (ghost) [reply test/home.http]: Fixed, now has a comment.

File: test/tag.http (hunk 7 above)
Comment (ghost) [reply test/tag.http]: Fixed, now has a comment.

File: test/tag.http (hunk 6 above)
Comment (ghost) [reply test/tag.http]: Fixed, now has a comment.

File: test/tag.http (hunk 5 above)
Comment (ghost) [reply test/tag.http]: Fixed, now has a comment.

File: test/profiles.http (hunk 4 above)
Comment (ghost) [reply test/profiles.http]: Fixed, now has a comment.

File: test/profiles.http (hunk 3 above)
Comment (ghost) [reply test/profiles.http]: Fixed, now has a comment.

File: test/profiles.http (hunk 2 above)
Comment (ghost) [reply test/profiles.http]: Fixed, now has a comment.

File: test/profiles.http (hunk 1 above)
Comment (ghost) [reply test/profiles.http:2]: Fixed, now has a comment.

**Pull request merged**